pytorch_lightning/trainer/distrib_data_parallel.py

```diff
@@ -163,6 +165,10 @@ def train_fx(trial_hparams, cluster_manager, _):
 else:
     XLA_AVAILABLE = True
 
+pid = os.getpid()
+rng1 = np.random.RandomState(pid)
+RANDOM_PORTS = rng1.randint(10000, 19999, 100)
```

Comment:
does this cause a failure for a distributed cluster > 100 nodes?

Reply:
the random_port thing is only used in non-multi node ddp.